### ISSUE TITLE
Package name format for postgresql 9.2 on CentOS 7 is "postresql" not "postgresql92"

### ISSUE THREAD

**Issue by gene1wood** [hw-cookbooks/postgresql#313]
In CentOS 7 it looks like the package name for postgresql 9.2.14 is `postgresql`. This can be seen in the [list of packages in the CentOS 7 updates repo](http://mirror.centos.org/centos/7/updates/x86_64/Packages/)

The [default attributes](https://github.com/hw-cookbooks/postgresql/blob/1ed6ab759d1a2474026c2c5eb7bb45c11c66e963/attributes/default.rb#L166-L168) however appear to assume that for CentOS 6 and CentOS 7, the package name format is `postgresqlXX` where `XX` is the version (example : `postgresql92`)

This problem results in chef failing with the error

```
No version specified, and no candidate version available for postgresql92-devel
```

I suspect that in CentOS 6 the package name format may have been `postgresql92` for versions other than `8.4` and in CentOS 7 it's now `postgresql` for version `9.2` in which case that conditional linked above should probably be tweaked.

It looks like [the code](https://github.com/hw-cookbooks/postgresql/blob/6584a0c05572b52992723705a4afc827e63016ba/attributes/default.rb#L161) is assuming that in CentOS 6, postgresql 8.4 has a package name of `postgresql`. Maybe add a case for CentOS7 and postgresql 9.2 the package name is also `postgresql`

**Comment by slyness** [comment]
Yes. This is all a known issue. I have a working branch which will be working better. I hope to release it before the new year. See #312

**Comment by slyness** [comment]
Resolved as of release 4.0.0 - https://github.com/hw-cookbooks/postgresql/releases/tag/v4.0.0
